src/platforms/unity/configuration/il2cpp.mdx

```diff
@@ -35,3 +33,5 @@ You can opt in to this feature through the editor configuration window by select
 ```csharp
 options.Il2CppLineNumberSupportEnabled = true;
 ```
+
+The support for line numbers in C# Exceptions with IL2CPP relies on the [debug symbol upload](/platforms/unity/configuration/il2cpp/). With the automated debug symbol upload enabled, the [Sentry SDK will upload the line mapping automatically](/platforms/unity/native-support/#debug-symbols).
```

Comment:
```suggestion
The support for line numbers in C# Exceptions with IL2CPP relies on the debug symbol upload. With the automated debug symbol upload enabled, the [Sentry SDK will upload the line mapping automatically](/platforms/unity/native-support/#debug-symbols).
```
Because otherwise, the site links to itself?